src/cli_callback.h

```diff
@@ -0,0 +1,179 @@
+/**
+ * @file storjapi_callback.h
+ * @brief Storj callback library.
+ *
+ * Implements callback functionality that can be customised for 
+ * end user's application
+ */
+
+#ifndef CLI_CALLBACK_H
+#define CLI_CALLBACK_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "storj.h"
+
+#if defined(_WIN32) && defined(STORJDLL)
```

Comment:
This DLL export stuff isn't needed here.

Reply:
removed the CLI_API references